lib/passkeys_signer.dart

```diff
@@ -0,0 +1,96 @@
+import 'dart:convert';
```

Comment:
can we separate the sdk and the example app?

Reply:
Yes, I'm planning to do that and publish the lib in the next PR